src/main/java/housing/Region.java

```diff
@@ -50,6 +61,21 @@ public Region(int targetPopulation) {
         regionalHouseholdStats = new RegionalHouseholdStats(this);
         regionalHousingMarketStats = new RegionalHousingMarketStats(houseSaleMarket);
         regionalRentalMarketStats = new RegionalRentalMarketStats(regionalHousingMarketStats, houseRentalMarket);
+        failedBidderCounter=new HashMap<Id<Household>,Integer>();
+    }
+    
+    /**
+     * @author HiTony: we need a ID for each region in order to make them distinguishable. 
+     */
+    public Region(int targetPopulation,Id<Region> regionId,Transport transport) {
```

Comment:
If this way of counting failed bids ends up prevailing, then we should remove the old constructor and keep only this one. Am I right?

Reply:
Yes, you are right. I will remove the old one.

Reply:
Ok, thanks.